src/main/arm/createUiDefinition.json

```diff
@@ -22,91 +31,130 @@
         ],
```

Comment:
At the end of the basics tab is where we need to insert the links requested in https://github.com/WASdev/azure.websphere-traditional.cluster/issues/98 . However, we don't have the links yet. So I suggest we put placeholder text in as follows.


- If you encounter problems during deployment of Liberty, please report them here [Issue tracker](ibm.biz link to the appropriate issue tracker)
- If you want to interact directly with the Liberty community, join the public Gitter. [Join gitter](ibm.biz link to join the gitter)
- To get free help with Azure migration from the development team, fill out this survey. [Take survey](https://aka.ms/migration-survey)

Reply:
Fixed in commit in a6499c2 by reusing changes in PR https://github.com/WASdev/azure.liberty.aro/pull/21.